db/migrate/20200802175827_remove_string_columns_that_should_be_floats.rb

```diff
@@ -0,0 +1,14 @@
+class RemoveStringColumnsThatShouldBeFloats < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :weathers, :high
+    remove_column :weathers, :low
+    remove_column :weathers, :feels_like
+    remove_column :weathers, :humidity
+    remove_column :weathers, :visibility
+    remove_column :weathers, :temp
+    remove_column :hourlies, :temp
+    remove_column :dailies, :high
+    remove_column :dailies, :low
```

Comment:
would have liked one migration to chagne the type, but I was getting errors. remove and create it is